src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -677,17 +674,20 @@ public TeamDetailsBundle getTeamDetailsForStudent(StudentAttributes student) {
     }
 
     /**
-     * lines is the enrollment lines entered by the instructor.
+     * Returns the formatted string for duplicate email error using {@code linesArray},
+     * the index of the first occurrence of the duplicate {@code isDuplicate}
+     * and sanitized enrollment line {@code line}.
      */
-    private String addDuplicateEmailInfo(String sanitizedLine, String[] linesArray, int isDuplicate) {
+    private String duplicateEmailInfo(String line, String[] linesArray, int isDuplicate) {
         String info =
-                getInvalidityInfoInDuplicatedEmail(linesArray, isDuplicate)
+                "Same email address as the student in line \"" + linesArray[isDuplicate + 1] + "\""
```

Comment:
1. Rename `isDuplicate`. I'm sure I mentioned it previously.

1. This String is a magic string: let's extract it out as a constant. As to where you should extract it out to (in this class? Or in `Const.StatusMessages`? Or elsewhere?), I'll leave it to you to figure it out.

Reply:
Done